server/app.ts

```diff
@@ -88,6 +89,23 @@ new MathigonStudioApp()
     const courses: TocCourse[] = tocFilterByType(type)
     res.json(courses)
   })
+  .get('/delete/account', async (req, res) => {
+    if (!req.user) return {error: 'unauthenticated', errorCode: 401, redirect: '/signin'}
+
+    const alphabet = 'abcdefghijklmnopqrstuvwxyz'
+    // 21 value is to maintain the probability collision similar to UUIDv4
+    const nanoid = customAlphabet(alphabet, 21)
+    const randomString = nanoid()
+    req.user.email = `deleted-${randomString}@qiskit.com`
```

Comment:
I know this is a fake email -- so I'm wondering if the `@qiskit.com` part matters... or should we be using `@qiskit.org`